### ISSUE TITLE
how to use ssim like ssim()

### ISSUE THREAD

**Issue by isalirezag** [scikit-image/scikit-image#2796]
I have seen people use 
`loss = nn.BCELoss()` to define their loss function.
Is there any way that I can use 
`loss = ssim()`.
I followed the skimage tutorial/example and it works. but I am not sure how I can define my loss like that. 
if I define `loss = ssim()`, it gives me an error that 

>  compare_ssim() takes at least 2 arguments (0 given)

Thank you for your help in advance

**Comment by stefanv** [comment]
Sorry, these issues are for tracking technical problems related to skimage only.  Please ask for help on the mailing list, if your question is scikit-image related (I don't know what nn is, so can't help there).
